internal/session/session.go

```diff
@@ -123,10 +121,13 @@ type Session struct {
 	consecutiveTicks    int
 	initialSearchDelay  time.Duration
 	periodicSearchDelay delay.D
+	wantRequests        map[*bswrm.WantRequest]struct{}
```

Comment:
The session now keeps track of individual WantRequests (each call to GetBlocks() creates a new WantRequest)